docs/app/Examples/elements/Divider/Types/DividerVerticalExample.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react';
+import {Segment, Divider} from 'stardust';
+
+export default class DividerVerticalExample extends Component {
+  render() {
+    return (
+      <div className='ui three column relaxed grid'>
+        <div className='column'>
+          <Segment className='basic'>
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Donec odio.
+          </Segment>
+        </div>
+        <Divider className='vertical'>Or</Divider>
+        <div className='column'>
+          <Segment className='basic'>
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Donec odio.
+          </Segment>
+        </div>
+        <Divider className='vertical'>And</Divider>
+        <div className='column'>
+          <Segment className='basic'>
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Donec odio.
+          </Segment>
+        </div>
+      </div>
+    );
+  }
+}
```

Comment:
Does SD have a `<Column>` component? I used regular `<div> and set the className to column in this example. We do need the columns though to get the proper formatting to show the vertical dividers